kqueen/server.py

```diff
@@ -1,4 +1,5 @@
 from flask import Flask
+from flask.ext.babel import Babel
```

Comment:
Can we do same with `python-dateutil` and `pytz`?

Reply:
Not with the same functionality. But if you want to keep backend on UTC always, there has to be date parsing and transformation in API client in the UI, so I might as well only return the UTC timestamp and move the smart stuff to UI.